src/app/features/settings/network/components/network-list-item-menu.tsx

```diff
@@ -37,6 +38,7 @@ export function NetworkItemMenu({ onClickDeleteNetwork, onEditNetwork }: Props)
               </HStack>
             </DropdownMenu.Item>
             <DropdownMenu.Item
+              data-testid={NetworkSelectors.DeleteNetworkMenuBtn}
```

Comment:
Is this UI right? Red text and black icon?

![Screenshot 2024-09-25 at 09 32 46](https://github.com/user-attachments/assets/1862396e-33b2-4307-9fd8-5d98fc3b6df9)